ColorPicker/ColorPicker.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * Agate component to allow the user to choose a color.
+ *
+ * @example
+ * <ColorPicker
+ *   spacing="medium"
+ * >
+ *   A group of related components
```

Comment:
It seems like `ColorPicker` doesn't handle children, maybe update the doc to illustrate `ColorPicker` is used as a self-closing component.

Reply:
The docs aren't updated to reflect the component. I was hoping to save time by getting feedback on the functionality and approach before filling in all of the documentation. I'll update the docs now though.

Reply:
Pretty much the only feedback I have because I was trying to figure out how to use it at a glance, otherwise it looks good!